Gemfile

```diff
@@ -203,3 +203,17 @@ end
 GemfileHelper.parse_each_agent_gem(ENV['ADDITIONAL_GEMS']) do |args|
   gem *args
 end
+
+if ENV['HUGINN_AGENT_SPEC_RUNNER']
+  base_path = File.expand_path('../../../', __FILE__)
```

Comment:
@cantino this can go in `GemfileHelper`. I tried to add a bit more explanation as to why I think the patch should live in `huginn/Gemfile` as opposed to `spec_runner.rb`:
https://github.com/cantino/huginn_agent/pull/15#issuecomment-287953886

I could also see this as being generic "additional gemspec" behavior outside of `huginn_agent`, e.g. `ADDITIONAL_GEMSPEC`